sdk/python/feast/infra/offline_stores/file.py

```diff
@@ -181,6 +181,8 @@ def evaluate_historical_retrieval():
                     inplace=True,
                 )
 
+                print(f"df_to_join: {df_to_join.columns}")
```

Comment:
This logging statement seems very cryptic. What should the user's takeaway be here?

Reply:
Also, should we be using a proper logger here?

Reply:
ignore it, I had an issue with columns being out of order and had to figure out what was going on.